src/ClientFlags/ClientFlags.cpp

```diff
@@ -55,7 +55,8 @@ ABSL_FLAG(bool, enforce_full_redraw, false,
 
 // VSI
 ABSL_FLAG(std::string, connection_target, "",
-          "Instance and process in the form <pid>@<instance_id>. Specify this to skip the "
+          "Instance and process in the form <pid>@<instance>, where <instance> is either an "
+          "instance ID or display name. Specify this to skip the "
```

Comment:
```suggestion
          "instance ID or a display name. Specify this to skip the "
```
Maybe?

Reply:
I think it's better without the "a", it's "instance ID or (instance) display name"

Reply:
This is comment is purely for fun, there's no reason to make any change, and that I'm most likely wrong! 
In my understanding:
"an" refers to "ID", because "instance" is an attributive "noun", hence why the change I suggested: "an instance ID or a(n instance) display name".
What you say would apply with the genitive: "an instance's ID or display name".

Reply:
Genitive feels weird as well... but I agree that this may be a bit unclear since "instance ID" is a somewhat fixed term. I've changed it to "an instance ID or an instance display name", even though it's more verbose it's at least clear what this is referring to.